core/embed/rust/src/ui/model_tr/mod.rs

```diff
@@ -7,11 +7,19 @@ pub mod component;
 pub mod constant;
 #[cfg(feature = "micropython")]
 pub mod layout;
-pub mod screens;
+mod screens;
 pub mod theme;
 
-pub struct ModelTRFeatures {}
+pub struct ModelTRUIFeaturesCommon {}
```

Comment:
Why not keep one struct (per model) and implement both traits for it?

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/584d8d209c049bd4ad6bedd674ec5f797dd28f2e